### DISCUSSION TITLE
SDK: Create - Add ability to create more than one item at a time

### DISCUSSION THREAD

**Discussion by RantingHuman** [mnfst/llm-gateway#125] [Feature request]
It would be useful if there was a feature to create multiple items at once, like a bulk create.

**Comment by brunobuddy** [comment]
Hello @RantingHuman, it can be interesting ! We are going to see if we put it in the roadmap

**Comment by brunobuddy** [comment]
Closing this out. It's from Manifest's old backend-as-a-service days, which we don't work on anymore since the pivot. See #742 for where things are now.
